### DISCUSSION TITLE
网上有些帖子说同时训练多个模型会影响训练质量，是否属实？

### DISCUSSION THREAD

**Discussion by TQG1997** [svc-develop-team/so-vits-svc#298] [Q&A]

**Comment by ylzz1997** [comment]
单个模型训练多个人物确实会影响模型质量，建议一个模型只训练一个人

**Comment by zengchang233** [comment]
@ylzz1997  你好，那如果想做multi singer的话，考虑到音质和自然度，那需要每个singer都训练一个模型吗？

**Comment by ylzz1997** [comment]
> @ylzz1997 你好，那如果想做multi singer的话，考虑到音质和自然度，那需要每个singer都训练一个模型吗？

如果考虑音质的话，最好是一个模型只训练一个

**Comment by PriesiaMioShirakana** [comment]
同时训练多个模型并不会影响任何东西，请不要相信网上的这些言论。
只有在同一个模型中训练多个角色（多角色模型）才有有着些许的影响

**Comment by Jared-02** [comment]
> 同时训练多个模型并不会影响任何东西，请不要相信网上的这些言论。 只有在同一个模型中训练多个角色（多角色模型）才有有着些许的影响

请问如果我只有该角色的谈话干声，单独训练出来的音域太窄，唱歌经常哑音、漏电。补充公开数据集中音色类似的唱歌干声进去是不是优化音域的最好办法呢？
我测试补充了 2 位 m4singer 数据集中的男声，每位接近 2.5h，总计不到 5h。加上原数据集中的 1.5h 谈话男声。3 个 speaker 在同一模型中训练，51k step 后，虽然有轻微的音色泄露，但是音域得到极大提升。请问我这种方法是否是可行的，或是您有什么更好的解决思路？

**Comment by HuanLinOTO** [comment]
it's look lite a problem of using, concert to discussion
